docs/tutorials/jokes.md

```diff
@@ -1620,12 +1620,13 @@ Remix and the `tsconfig.json` you get from the starter template are configured t
 
 <summary>app/routes/jokes.tsx</summary>
 
-```tsx filename=app/routes/jokes.tsx lines=[3,5,12,19-21,23-28,31,55-59]
+```tsx filename=app/routes/jokes.tsx lines=[3,4,6,12,19-21,23-28,31,55-59]
```

Comment:
Needs to be reverted, see next comment

Reply:
@machour, thanks for your feedbacks ser - had reflected them in recent commits
* separated `json` import from `types` import
* imported `Joke` type on `findMany` example - placed directly below `import type`'s for grouping (updated line highlight for new lines added)
* removed `Pick` from projected example and replaced with `{ id:string, name:string }` - add line highlight so reader is aware that original `Joke` type needs to be edited